components/Game/WinLoseScreen.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from "react";
+import "react-native-gesture-handler";
+import { useWindowDimensions } from "react-native";
+import styled from "styled-components/native";
+import { NavigationContainer } from "@react-navigation/native";
+
+export const WinLoseScreen = ({ route, navigation }) => {
+  const { hero, villain, winner } = route.params;
+  const { width: screenWidth, height: screenHeight } = useWindowDimensions();
+
+  return (
+    <>
```

Comment:
You have only one scroll component returned here (the rest of them is inside of it) so react fragment is not needed here.